Examples/Swift/UserLocationAnnotationExample.swift

```diff
@@ -81,14 +82,13 @@ class CustomUserLocationAnnotationView: MGLUserLocationAnnotationView {
             arrowSize = size / 2.5
             arrow = CAShapeLayer()
             arrow.path = arrowPath()
-            arrow.bounds = CGRect(x: 0, y: (size/2 + 2), width: arrowSize, height: arrowSize)
+            arrow.bounds = CGRect(x: 0, y: 0, width: arrowSize, height: arrowSize)
```

Comment:
The arrow never appears after this change, because it has no `position`. Going back to a suggestion @1ec5 made long ago in https://github.com/mapbox/ios-sdk-examples/pull/30#discussion_r92530800, it would probably be better to set a frame (and not have to worry about bounds+position.)